chromium_src/third_party/blink/renderer/modules/storage/dom_window_storage.cc

```diff
@@ -209,18 +210,10 @@ StorageArea* BraveDOMWindowStorage::localStorage(
   LocalDOMWindow* window = GetSupplementable();
   auto* storage = DOMWindowStorage::From(*window).localStorage(exception_state);
 
-  MaybeClearAccessDeniedException(storage, *window, &exception_state);
-  if (!base::FeatureList::IsEnabled(net::features::kBraveEphemeralStorage))
+  if (!ShouldUseEphemeralStorage(
+          window, WebContentSettingsClient::StorageType::kSessionStorage))
```

Comment:
Should this be `WebContentSettingsClient::StorageType::kLocalStorage` instead? If not, can the argument be eliminated from the function entirely since each call site passes `WebContentSettingsClient::StorageType::kSessionStorage`?

Reply:
Good catch. It won't affect anything because the ephemeral storage settings are the same for session and local storage, but it should be correct in case something changes with that in the future